.github/workflows/lint-pr.yml

```diff
@@ -6,6 +6,7 @@ on:
       - opened
       - edited
       - synchronize
+  merge_group:
```

Comment:
Doesn't this need to specify branches, like in the other actions? E.g, https://github.com/cloudscape-design/components/blob/main/.github/workflows/build-lint-test.yml#L8